include/keyence/impl/messages/prepare_high_speed.h

```diff
@@ -0,0 +1,52 @@
+#ifndef PREPARE_HIGH_SPEED_H
+#define PREPARE_HIGH_SPEED_H
+
+#include "../keyence_message.h"
```

Comment:
Is there a reason you chose to use relative references here?  Why not include paths relative to the package include?